vertx/pom.xml

```diff
@@ -24,16 +24,22 @@
     <version>13.6-SNAPSHOT</version>
   </parent>
 
-  <artifactId>feign-vertx</artifactId>
+  <artifactId>feign-vertx-parent</artifactId>
+  <packaging>pom</packaging>
 
-  <name>Feign Vertx</name>
-  <description>Implementation of Feign on Vertx web client.</description>
+  <modules>
+    <module>feign-vertx</module>
+    <module>tests-vertx4</module>
+    <module>tests-vertx-common</module>
+    <module>tests-vertx5</module>
```

Comment:
nice effort on test coverage 🥇

Reply:
I think we can just call them `feign-vertx*-test`

Reply:
Thanks for the suggestion. I will study them carefully and see how I can improve tests. The most important is to avoid to produce the jar and run tests at same build. By doing so we will not be able to detect runtime binary uncompatibilities with different versions of Vertx (that arise quite offten).

Reply:
> nice effort on test coverage 🥇

Thanks! feign-vertx is a 10-year-old project. It had plenty of time to be tested in productions and covered with tests for the most of frequent problems.

Reply:
> I think we can just call them `feign-vertx*-test`

done